definitions/v7/lat-team-api.yml

```diff
@@ -113,24 +107,12 @@ search:
       selector: name:contains(VOSE)
       optional: true
       filters:
-        - name: re_replace
-          args: ["^ *\\[[^\\]]*\\] *", ""] # Remove prefix tags
-        - name: re_replace
-          args: ["(?i)\\bT(\\d+)", "S$1"]
-        - name: re_replace
-          args: ["UHDRip", "BDRip"] # Fix for Radarr
```

Comment:
this is for parsing the results returned

This is a non-issue anymore and all previous UHDRip on the site were changed to BDRip?

Reply:
There are no, and never will be, UHDRip names on the site.